Sharpmake.Generators/VisualStudio/Csproj.Template.cs

```diff
@@ -113,6 +113,7 @@ public static class Project
     <ProductVersion>[options.ProductVersion]</ProductVersion>
     <UseWpf>[options.UseWpf]</UseWpf>
     <UseWindowsForms>[options.UseWindowsForms]</UseWindowsForms>
+    <WarningsAsErrors>[options.WarningsAsErrors]</WarningsAsErrors>
```

Comment:
Shouldn't this be in the ProjectConfigurationsGeneral section below with the other warning related properties?

Reply:
Yes, absolutely